exercises/camicia/instructions.md

```diff
@@ -0,0 +1,58 @@
+# Instructions
+
+In this exercise, you will simulate the classic card game **Beggar-my-neighbour** (*Camicia*).
+Your program will receive the initial configuration of two players’ decks and must simulate the game until it ends (or detect if it never does).
+
+## Rules
+
+* The deck is split between **two players**. Each player’s cards are represented as a string (top of the deck = leftmost character).
+
+* Players take turns placing the **top card** of their deck onto a central pile.
+
+* If the card is a **number card** (`-` in this scenario), play simply passes to the other player.
+
+* If the card is a **payment card**:
+
+  * `J` → opponent must pay 1 card
+  * `Q` → opponent must pay 2 cards
+  * `K` → opponent must pay 3 cards
+  * `A` → opponent must pay 4 cards
+
+* While paying, if the opponent reveals another payment card, the penalty immediately switches back.
```

Comment:
```suggestion
* While paying, if the opponent reveals another payment card, the price to be paid immediately switches back to the original player.
```

Reply:
Note, the play paying a price can switch multiple times. It doesn't always switch to the "original" player.
```suggestion
* If the player paying a penalty reveals another payment card, that player stops paying the penalty and the other player much then pay the penalty.
```